JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -5,16 +5,118 @@
 // 
 
 import UIKit
-import Foundation
 
-class ViewController: UIViewController {
+extension Notification.Name {
+    static let recieveOrderMenuNotification = Notification.Name("DidRecieveOrderMenu")
+    static let recieveStockChangeNotification = Notification.Name("DidReceiveStockChange")
+}
 
+class ViewController: UIViewController {
+    let juiceMaker = JuiceMaker()
+    
+    @IBOutlet weak var strawberryStockLabel: UILabel!
+    @IBOutlet weak var bananaStockLabel: UILabel!
+    @IBOutlet weak var fineappleStockLabel: UILabel!
+    @IBOutlet weak var kiwiStockLabel: UILabel!
+    @IBOutlet weak var mangoStockLabel: UILabel!
+    
+    @IBOutlet weak var strawberryJuiceButton: UIButton!
+    @IBOutlet weak var bananaJuiceButton: UIButton!
+    @IBOutlet weak var kiwiJuiceButton: UIButton!
+    @IBOutlet weak var fineappleJuiceButton: UIButton!
+    @IBOutlet weak var strawberryBananaJuiceButton: UIButton!
+    @IBOutlet weak var mangoJuiceButton: UIButton!
+    @IBOutlet weak var mangoKiwiJuiceButton: UIButton!
```

Comment:
```suggestion
    @IBOutlet private weak var strawberryStockLabel: UILabel!
    @IBOutlet private weak var bananaStockLabel: UILabel!
    @IBOutlet private weak var fineappleStockLabel: UILabel!
    @IBOutlet private weak var kiwiStockLabel: UILabel!
    @IBOutlet private weak var mangoStockLabel: UILabel!
    
    @IBOutlet private weak var strawberryJuiceButton: UIButton!
    @IBOutlet private weak var bananaJuiceButton: UIButton!
    @IBOutlet private weak var kiwiJuiceButton: UIButton!
    @IBOutlet private weak var fineappleJuiceButton: UIButton!
    @IBOutlet private weak var strawberryBananaJuiceButton: UIButton!
    @IBOutlet private weak var mangoJuiceButton: UIButton!
    @IBOutlet private weak var mangoKiwiJuiceButton: UIButton!
```

@IBOutlet도 외부에서 알 필요가 없으면 private을 붙여도 좋을 것 같습니다.